builds/linux/almalinux/8/variables.pkr.hcl

```diff
@@ -42,6 +42,11 @@ variable "vsphere_cluster" {
   description = "The name of the target vSphere cluster. (e.g. 'sfo-w01-cl01')"
 }
 
+variable "vsphere_host" {
+  type        = string
+  description = "The name of the target vSphere host. (e.g. 'sfo-w01-esx01')"
```

Comment:
“ESXi host” is the correct product term.

Reply:
Modified the descriptions, think that the variable name should stay `vsphere_host` for consistency with the other vSphere related variables.